lib/manager/terraform/extract.ts

```diff
@@ -28,7 +28,7 @@ import {
 } from './util';
 
 const dependencyBlockExtractionRegex = regEx(
-  /^\s*(?<type>[a-z_]+)\s+("(?<lookupName>[^"]+)"\s+)?("(?<terraformName>[^"]+)"\s+)?{\s*$/
+  /^\s*(?P<type>[a-z_]+)\s+("(?P<lookupName>[^"]+)"\s+)?("(?P<terraformName>[^"]+)"\s+)?{\s*$/
```

Comment:
Nope, this won't fix it. Maybe the `{` needs to be escaped?

Reply:
Have copied and made another PR #12778.
I have tried not to disturb any terraform regex without good reason and also kept `ensureTrailingSlash` and `trimLeadingSlash` out of the picture for now since we can look at them in this issue #12071.

Reply:
I can't add a suggestion here, please revert the `P` change and escape the last `{`

`/^\s*(?<type>[a-z_]+)\s+("(?<lookupName>[^"]+)"\s+)?("(?<terraformName>[^"]+)"\s+)?\{\s*$/`

I think that will fix the issue. This should also converted to a string instead of regex.